src/main/java/com/researchspace/webapp/filter/SAMLRemoteUserPolicy.java

```diff
@@ -1,24 +1,18 @@
 package com.researchspace.webapp.filter;
 
-import com.researchspace.model.permissions.SecurityLogger;
-import java.util.Enumeration;
 import java.util.Map;
 import java.util.TreeMap;
 import javax.servlet.http.HttpServletRequest;
 import org.apache.commons.lang3.StringUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /** Gets the Shibboleth remote username based on HttpHeader information. */
-public class SAMLRemoteUserPolicy implements RemoteUserRetrievalPolicy {
+public class SAMLRemoteUserPolicy extends AbstractSsoRemoteUserPolicy {
 
   protected static final String SHIBBOLETH_ID_ATTRIBUTE = "eduPersonPrincipalName";
 
-  private static final Logger log = LoggerFactory.getLogger(SecurityLogger.class);
-
-  // don't alter, edit or remove these as they may be used in production deployments
-  // only add new values if needed.
-  static final String[] shib_attributes = {
+  /* don't alter, edit or remove these as they may be used in production deployments;
+  only add new values if needed. */
+  private static final String[] shib_attributes = {
```

Comment:
We could add a test to make sure that the number of values and the values themselves are not mistakely changed

Reply:
Yes good point, but let's maybe leave it for next person changing that SAML attributes list ;) 
(I was only reformatting the comment with my commit here).